src/org/zaproxy/zap/extension/params/SiteParameters.java

```diff
@@ -93,21 +96,21 @@ public HtmlParameterStats getParam(HtmlParameter.Type type, String name) {
 				params.add(param);
 			}
 			break;
+		case header:
+			for (HtmlParameterStats param : this.headerParams.values()) {
+				params.add(param);
```

Comment:
This could also use the `addAll` (same for above cases), as done in `getParams()`.